strings/names/last.txt

```diff
@@ -528,7 +528,7 @@ Warren
 Watson
 Webb
 Weeter
-Weinstein
```

Comment:
how bad is this?

Reply:
It just references a very famous american film producer, he worked on all 3 LOTR movies.